src/pages/ApplicationList/index.tsx

```diff
@@ -110,10 +124,45 @@ class Application extends Component<Props, State> {
     });
   };
 
+  isEditPlatForm = () => {
+    const { userInfo } = this.state;
+    const isAdminUser = this.isAdminUserCheck();
+    if (isAdminUser && userInfo && !userInfo.email) {
+      this.setState({
+        isEditAdminUser: true,
+      });
+    }
+  };
+
+  isAdminUserCheck = () => {
```

Comment:
The util function mv to the util package.

Reply:
done